gitpage/app/controllers/gitpage_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GitpageController < ApplicationController
+  def index
+    @profile = HTTParty.get('https://api.github.com/search/users?q=cdmackeyfree')
+    @repos = HTTParty.get(https://api.github.com/users/cdmackeyfree/repos?sort=updated)
+  end
```

Comment:
the url on line 4 needs to be in quotation marks -- it will throw an error as is